concrete/single_pages/account/messages.php

```diff
@@ -244,12 +244,12 @@
 
 			<table class="table table-striped" border="0" cellspacing="0" cellpadding="0">
 			<tr>
-				<th class="ccm-profile-messages-item-name"><?=t('Mailbox')?></th>
```

Comment:
I feel like this page is likely to be customized with CSS in a site and removing this might have consequences. Should we keep this class and add the new one + some css to override the 100% width?